cmd/lookoutd/web.go

```diff
@@ -21,12 +21,17 @@ type WebCommand struct {
 	ServerURL          string `long:"server" env:"LOOKOUT_SERVER_URL" description:"URL used to access the web server in the form 'HOSTNAME[:PORT]'. Leave it unset to allow connections from any proxy or public address"`
 	FooterHTML         string `long:"footer" env:"LOOKOUT_FOOTER_HTML" description:"Allows to add any custom html to the page footer. It must be a string encoded in base64. Use it, for example, to add your analytics tracking code snippet"`
 
+	ClientID     string `long:"client-id" env:"LOOKOUT_CLIENT_ID" required:"true" description:"GitHub App OAuth credentials"`
```

Comment:
I think we could put these settings in the config file.
We already have other attributes of the GitHub App there, so it makes sense for consistency. When we integrate the `web` part into the `serve` sub command it will be strange to have some settings as options, and others in the config file.

Reply:
I was thinking more about k8s deployment. 

In `serve` we actually already have some settings as options and some in config file.
`--github-user` and `--github-token` should be passed as options.

I agree it looks a bit messy. Let me think about it a bit.

Reply:
After giving it some thoughts I feel like we are mixing 2 different ways how lookoutd is suppose to be used:
1. Run locally using `serve` command.
This case includes using github user/token and manual repositories configuration.
There is no need to use queue or github app. lookout-web wouldn't work in such case at all. 
And it's very handy to use yml config file.

2. Run lookoutd as a service.
It means distributed mode, queue and github app. For this, it definitely makes sense to configure `watch`/`work`/`web` using standard env variables that are much easier to deal with in most of the deployment systems like k8s.
The only problem is the list of analyzers. If we implement service discovery next year then this issue will be solved and we can apply standard configuration approach.

For now without service discovery I don't see much value in env vars and I'll update PR using yaml config instead.

Reply:
Makes sense. If I recall correctly, the list of analyzers was the exact need that made us go for a config file.